applications/si-front-end-template.md

```diff
@@ -0,0 +1,148 @@
+
+The team that was previously assembled is no longer active, and this grant is to be completed as private person.
+
+### Team Code Repos
+
+https://github.com/stojanov-igor
+
+### Team LinkedIn Profiles
+
+https://www.linkedin.com/in/igor-stojanov-96364ba/
+
+
+## Development Status :open_book:
+
+
+## Development Roadmap :nut_and_bolt:
+
+
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 10,000 USD
+
+### Milestone 1  — Design & Initial code
+
+- **Estimated duration:** 1 month
+- **FTE:**  1
+- **Costs:** 5,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide **inline documentation** of the code |
+| **0c.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 2. | Menu Header | UI element that replaces existing menu items |
+| 3. | Footer | New Element for implementing a new UI Element footer |
+| 4. | Polkadot Extension| Ensure working connection with Polkadot extension |
+| 5. | Light/Dark Mode| Enable styling switch to dark and light mode |
+
+
+
+### Milestone 2  — Full implementation
+
+- **Estimated Duration:** 1 month
+- **FTE:**  1
+- **Costs:** 5,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide **inline documentation** of the code |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). (Content, language, and medium should reflect your target audience described above.) |
+| 1. | Cards Module | Implement dashboard cards with basic statistics about the blockchain. |
+| 2. | Table Module| Add table that holds Account balances that can be loaded from Polkadot Extension |
+| 3. | Transfer Module | Simple module to transfer tokens from one acount to another |
+| 4. | Upgrade Runtime Module| EEnsure users can upload wasm File from the UI |
+| 5. | Pallet Interactor | Module that allows interaction with extrinsics and queries |
+| 6. | Event Module | Displaying latest events that happens on-chain
```

Comment:
Are these new features?

Reply:
Hi @takahser ,

These are not new features, but UI redisin of existing modules.